### PR TITLE
Intel471 threat intel c2 c

### PR DESCRIPTION
**Pull request by amee-sumo**
## Purpose of this pull request

This pull request is to create a new doc for Intel 471 Threat Intel

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-583](https://sumologic.atlassian.net/jira/software/c/projects/DOCS/boards/516?assignee=712020%3Af4a4da3b-21a9-427c-8811-1b0a9f2e34f8&selectedIssue=DOCS-583)

### CODE REVIEW

File: cid-redirects.json [hunk 1]
```diff
@@ -1613,6 +1613,7 @@
   "/cid/10196": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zero-networks-segment-source",
   "/cid/10122": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source",
   "/cid/10125": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-cyber-resilience-source",
+  "/cid/10126": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source",
```
Comment (jpipkin1) [inline cid-redirects.json:1616]:
```suggestion
  "/cid/10127": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source",
```

File: cid-redirects.json [hunk 2]
```diff
@@ -1613,6 +1613,7 @@
   "/cid/10196": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/zero-networks-segment-source",
   "/cid/10122": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/webex-source",
   "/cid/10125": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-cyber-resilience-source",
+  "/cid/10127": "/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source",
```
Comment (jpipkin1) [inline cid-redirects.json:1616]: @amee-sumo, I changed the CID from "10126" to "10127" to resolve a merge conflict.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md [hunk 3]
```diff
@@ -295,6 +295,12 @@ In this section, we'll introduce the following concepts:
   <p>Collects a list of users from the Google Workspace Users API.</p>
   </div>
 </div>
+<div className="box smallbox card">
+  <div className="container">
+  <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source"><img src={useBaseUrl('img/send-data/intel471-threat-intel.png')} alt="thumbnail icon" width="180"/><h4>Intel 471 Threat Intel</h4></a>
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md:300]:
```suggestion
  <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source"><img src={useBaseUrl('img/send-data/intel471-threat-intel.png')} alt="thumbnail icon" width="100"/><h4>Intel 471 Threat Intel</h4></a>
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+:::info
+The Intel 471 API documentation is not public and can only be accessed by partners or customers.
+:::
+
+The Intel 471 Threat Intel source requires you to authenticate the **Username** and **API Key** found in the API section in the [portal](https://login.intel471.com/u/login/identifier?state=hKFo2SB4ckUwei1aN2JUaFhkTG5FQkstbF9sZldMTlFzdnZXMaFur3VuaXZlcnNhbC1sb2dpbqN0aWTZIG5HU3o1RVg3RTY2RWloTGFySjNTcWowLWVXRVJzYUxIo2NpZNkgR1dKRFJFclBoS3BRZGlWcmxNN2F5c0V6TDZXeEw5azI#section/Authentication).
+
+### Source configuration
+
+When you create an Intel 471 Threat Intel source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
```
Comment (jpipkin1) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:41]:
```suggestion
When you create an Intel 471 Threat Intel source, you add it to a Hosted Collector. Before creating the source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector and Source](/docs/send-data/hosted-collectors/configure-hosted-collector).
```

**Review by jpipkin1 — APPROVED** [review]
Looks great, @amee-sumo! I have only a few suggestions. Also, I changed the CID from 10126 to 10127 to fix a merge conflict, and I added the new source to the product list (see commit "[Add to product list](https://github.com/SumoLogic/sumologic-documentation/pull/4845/commits/2ce7efcb2a13b226a2c1f43d14583d14dd83d386)").

File: blog-service/2024-12-10-collection.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+title: Intel 471 Threat Intel Source (Collection)
```
Comment (JV0812) [inline blog-service/2024-12-10-collection.md:2]:
```suggestion
title: Intel471 Threat Intel Source (Collection)
```

File: blog-service/2024-12-10-collection.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  - collection
+  - intel471-threat-intel
+hide_table_of_contents: true    
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<a href="https://help.sumologic.com/release-notes-service/rss.xml"><img src={useBaseUrl('img/release-notes/rss-orange2.png')} alt="icon" width="50"/></a>
+
+We're excited to announce the release of our new cloud-to-cloud source for Intel 471 Threat Intel. This source helps you to collect threat indicators from the Intel 471 API and send them to Sumo Logic as normalized threat indicators for analysis. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source).
```
Comment (JV0812) [inline blog-service/2024-12-10-collection.md:14]:
```suggestion
We're excited to announce the release of our new cloud-to-cloud source for Intel471 Threat Intel. This source helps you to collect threat indicators using the [Intel471 Stream API](https://titan.intel471.com/api/docs-openapi/#tag/Indicators/paths/~1indicators~1stream/get) and send them to Sumo Logic for normalized analysis. [Learn more](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source).
```

File: docs/integrations/product-list/product-list-a-l.md [hunk 7]
```diff
@@ -284,7 +284,7 @@ For descriptions of the different types of integrations Sumo Logic offers, see [
...
 | <img src={useBaseUrl('img/integrations/misc/isc-logo.png')} alt="Thumbnail icon" width="50"/> | [ISC](https://www.isc.org/) | Cloud SIEM integration: [ISC](https://github.com/SumoLogic/cloud-siem-content-catalog/blob/master/vendors/1583cfd2-7ece-4060-991b-06dcf8567943.md) |
 |  <img src={useBaseUrl('img/integrations/saas-cloud/istio.png')} alt="Thumbnail icon" width="50"/>   | [Istio](https://istio.io/)  | 	App: [Istio](/docs/integrations/saas-cloud/istio/)	 |
-| <img src={useBaseUrl('img/platform-services/automation-service/app-central/logos/intel-471.png')} alt="Thumbnail icon" width="75"/>  | [Intel 471](https://intel471.com/) | Automation integration: [Intel 471](/docs/platform-services/automation-service/app-central/integrations/intel-471/) |
+| <img src={useBaseUrl('img/platform-services/automation-service/app-central/logos/intel-471.png')} alt="Thumbnail icon" width="75"/>  | [Intel 471](https://intel471.com/) | Automation integration: [Intel 471](/docs/platform-services/automation-service/app-central/integrations/intel-471/) <br/>Collector: [Intel 471 Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source)|
```
Comment (JV0812) [inline docs/integrations/product-list/product-list-a-l.md:287]:
```suggestion
| <img src={useBaseUrl('img/platform-services/automation-service/app-central/logos/intel-471.png')} alt="Thumbnail icon" width="75"/>  | [Intel471](https://intel471.com/) | Automation integration: [Intel471](/docs/platform-services/automation-service/app-central/integrations/intel-471/) <br/>Collector: [Intel471 Threat Intel Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source)|
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md [hunk 8]
```diff
@@ -295,6 +295,12 @@ In this section, we'll introduce the following concepts:
   <p>Collects a list of users from the Google Workspace Users API.</p>
   </div>
 </div>
+<div className="box smallbox card">
+  <div className="container">
+  <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source"><img src={useBaseUrl('img/send-data/intel471-threat-intel.png')} alt="thumbnail icon" width="100"/><h4>Intel 471 Threat Intel</h4></a>
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md:300]:
```suggestion
  <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel471-threat-intel-source"><img src={useBaseUrl('img/send-data/intel471-threat-intel.png')} alt="thumbnail icon" width="100"/><h4>Intel471 Threat Intel</h4></a>
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md [hunk 9]
```diff
@@ -295,6 +295,12 @@ In this section, we'll introduce the following concepts:
...
+  <p>Collects threat indicators from the Intel 471 API.</p>
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md:301]:
```suggestion
  <p>Learn to collect threat indicators from the Intel471 platform.</p>
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
+---
+id: intel471-threat-intel-source
+title: Intel 471 Threat Intel Source
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:3]:
```suggestion
title: Intel471 Threat Intel Source
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+sidebar_label: Intel 471 Threat Intel
+tags:
+  - cloud-to-cloud
+  - intel471-threat-intel
+description: This integration collects threat indicators from the Intel 471 API and sends them to Sumo Logic as normalized threat indicators for analysis.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:8]:
```suggestion
description: This integration collects threat indicators using the Intel 471 API and sends them to Sumo Logic  for analysis.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/intel471-threat-intel/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/intel471-threat-intel/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/intel471-threat-intel/example.tf';
+import ForwardToSiem from '/docs/reuse/forward-to-siem.md';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/send-data/intel471-threat-intel.png')} alt="intel471-threat-intel.png" width="100" />
+
+Intel 471 helps organizations protect against cyber threats by monitoring underground marketplaces and forums used by cybercriminals. They specialize in cyber threat intelligence, focusing on malware, malicious actors, their tactics, techniques, and procedures (TTPs).
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:19]:
```suggestion
Intel 471 is a cybersecurity firm specializing in providing cyber threat intelligence services. Their focus is primarily on delivering information about threats originating from the criminal underground, including malware, malicious actors, and their tactics, techniques, and procedures (TTPs). Intel 471 provides these insights to help organizations protect themselves against cyber threats. Their intelligence-gathering efforts often involve monitoring and analyzing underground marketplaces, forums, and other communication channels used by cyber criminals.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+
+The Intel 471 source collects threat indicators from the [Intel 471 API](https://login.intel471.com/u/login/identifier?state=hKFo2SBJbDlzQnBDWGo0MU5wZEJndGh0YV9Eby0yQmRHb0dueqFur3VuaXZlcnNhbC1sb2dpbqN0aWTZIDY3NW1vbmdfdFktTTVDY01kY1ZncndOSU9ZZllxSXBao2NpZNkgR1dKRFJFclBoS3BRZGlWcmxNN2F5c0V6TDZXeEw5azI#tag/Indicators/paths/~1indicators~1stream/get) and sends them to Sumo Logic as normalized threat indicators for analysis.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:21]:
```suggestion
The Intel 471 source collects threat indicators using the [Intel 471 Stream API](https://titan.intel471.com/api/docs-openapi/#tag/Indicators/paths/~1indicators~1stream/get) and sends them to Sumo Logic as normalized threat indicators for analysis.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+
+## Data collected
+
+| Polling Interval | Data |
+| :--- | :--- |
+| 1 hr | [Indicators](https://login.intel471.com/u/login/identifier?state=hKFo2SBnRnh4QWZIandtd2J2S2pOeGdWS1MwWWZ1bFpubzctR6Fur3VuaXZlcnNhbC1sb2dpbqN0aWTZIEZXOFR0UGtYNWJLV3lWWXY4ME1tRlVrWk0yd0tneFBFo2NpZNkgR1dKRFJFclBoS3BRZGlWcmxNN2F5c0V6TDZXeEw5azI#tag/Indicators) |
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:27]:
```suggestion
| 1 hour | [Indicators](https://titan.intel471.com/api/docs-openapi/#tag/Indicators/paths/~1indicators~1stream/get) |
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+
+## Setup
+
+### Vendor configuration
+
+:::info
+The Intel 471 API documentation is not public and can only be accessed by partners or customers.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:34]:
```suggestion
The Intel 471 documentation is not public and can only be accessed by partners or customers.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+:::
+
+The Intel 471 Threat Intel source requires you to authenticate the **Username** and **API Key** found in the API section in the [portal](https://login.intel471.com/u/login/identifier?state=hKFo2SB4ckUwei1aN2JUaFhkTG5FQkstbF9sZldMTlFzdnZXMaFur3VuaXZlcnNhbC1sb2dpbqN0aWTZIG5HU3o1RVg3RTY2RWloTGFySjNTcWowLWVXRVJzYUxIo2NpZNkgR1dKRFJFclBoS3BRZGlWcmxNN2F5c0V6TDZXeEw5azI#section/Authentication).
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:37]:
```suggestion
The Intel 471 Threat Intel source requires you to provide the **Username** and **API Key** found in the API section in the [Intel 471 portal](https://titan.intel471.com/api/docs-openapi/#tag/Indicators/paths/~1indicators~1stream/get).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 17]
```diff
@@ -0,0 +1,99 @@
...
+1. On the Collectors page, click **Add Source** next to a Hosted Collector.
+1. Search for and select **Intel 471 Threat Intel**.
+1. Enter a **Name** to display for the Source in the Sumo web application. The description is optional.
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields.** Click the **+Add Field** link to define the fields you want to associate, each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema it is ignored, known as dropped. 
+1. **Username**. Enter the login ID or email address of the user.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:53]:
```suggestion
1. **Username**. Enter your Intel 471 login ID or email address.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 18]
```diff
@@ -0,0 +1,99 @@
...
+1. **API Key**. Enter the API key of the user account collected from the Intel 471 Threat Intel platform.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:54]:
```suggestion
1. **API Key**. Enter the API key of the user account collected from the [Intel 471 Threat Intel platform](#vendor-configuration).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 19]
```diff
@@ -0,0 +1,99 @@
...
+1. **Sumo Logic Threat Intel Source ID**. Enter the Sumo Logic namespace where the indicators will be stored.
+1. **Polling Interval**. The polling interval is set for 5 minutes by default. You can adjust it based on your needs. This sets how often the source checks for new data.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:56]:
```suggestion
1. **Polling Interval**. The polling interval is set for one hour by default. You can adjust it based on your needs. This sets how often the source checks for new data.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 20]
```diff
@@ -0,0 +1,99 @@
...
+1. **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+1. When you are finished configuring the source, click **Save**.
+
+## JSON schema
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [Use JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
+
+| Parameter | Type | Value | Required | Description |
+|:--|:--|:--|:--|:--|
+| schemaRef | JSON Object  | `{“type”: “Intel471 Threat Intel”}` | Yes | Define the specific schema type. |
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:66]: Intel 471 or Intel471? Need confirmation with Parth.

File: static/img/integrations/security-threat-detection/intel471-threat-intel.png
Comment (JV0812) [inline static/img/integrations/security-threat-detection/intel471-threat-intel.png:1]: This image has been not used anywhere in this PR. Can you confirm if this is required or not?

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fixes. Also, confirm if we need to mention as` Intel 471` or `Intel471`, accordingly update the entire PR.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md (hunk 20 above)
Comment (amee-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:66]: Both are correct, but let's stick to "Intel471" based on the naming convention in our Sumo platform.

File: static/img/integrations/security-threat-detection/intel471-threat-intel.png
Comment (amee-sumo) [reply static/img/integrations/security-threat-detection/intel471-threat-intel.png:1]: I will replace the existing image with this one.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 21]
```diff
@@ -0,0 +1,99 @@
+---
+id: intel471-threat-intel-source
+title: Intel471 Threat Intel Source
+sidebar_label: Intel471 Threat Intel
+tags:
+  - cloud-to-cloud
+  - intel471-threat-intel
+description: This integration collects threat indicators using the Intel471 API and sends them to Sumo Logic  for analysis.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:8]:
```suggestion
description: This integration collects threat indicators using the Intel471 API and sends them to Sumo Logic for analysis.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md [hunk 22]
```diff
@@ -0,0 +1,99 @@
...
+tags:
+  - cloud-to-cloud
+  - intel471-threat-intel
+description: This integration collects threat indicators using the Intel471 API and sends them to Sumo Logic for analysis.
+---
+import CodeBlock from '@theme/CodeBlock';
+import ExampleJSON from '/files/c2c/intel471-threat-intel/example.json';
+import MyComponentSource from '!!raw-loader!/files/c2c/intel471-threat-intel/example.json';
+import TerraformExample from '!!raw-loader!/files/c2c/intel471-threat-intel/example.tf';
+import ForwardToSiem from '/docs/reuse/forward-to-siem.md';
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source.md:14]:
```suggestion
```

**Review by JV0812 — APPROVED** [review]
LGTM

**Comment by yleiferman** [comment]
Hi @amee-sumo and @JV0812 , thank you for creating this documentation. As of now, the threat intelligence functionality is in beta, so can you update the docs to not make it official yet? Please let me know if you have any questions.

Maybe @jpipkin1 can you help during our timezone? My plan is to enable it for beta customers today.

**Comment by jpipkin1** [comment]
@yleiferman, if this is beta functionality, the Intel471 Threat Intel Source article should be marked as beta, like the [STIX/TAXII 1 Client Source](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/stix-taxii-1-client-source/) and the  [STIX/TAXII 2 Client Source](https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/stix-taxii-2-client-source/) articles. I can update this PR to do that.

**Comment by jpipkin1** [comment]
@yleiferman, the article is now marked as beta, and will be published here:
https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/intel-471-threat-intel-source

**Pull request merged**